README.md

```diff
@@ -1 +1,16 @@
-# tree-sitter-crystal
+# Crystal Treesitter
+Hello! You're looking at an ongoing WIP [tree-sitter](https://github.com/tree-sitter/tree-sitter) language support project ;) for the [Crystal](https://crystal-lang.org/) language. **Why?** This is a part of a drive to increase Crystal tooling across workflows (myself using neovim), and specifically consolidate efforts in the Crystal Tree Sitter realm. See [this post](https://forum.crystal-lang.org/t/tree-sitter-crystal/3565) for more context on this repos creation.
+
+## Installation
```

Comment:
```suggestion
```

An installation guide can come later when the project is in a more stable state, there's not much people can use this for right now.